docs/USING_PRO.md

```diff
@@ -1,24 +1,71 @@
 ## Extending Marked
 
-To champion the single-responsibility and open/closed principles, we have tried to make it relatively painless to extend marked. If you are looking to add custom functionality, this is the place to start.
+To champion the single-responsibility and open/closed principles, we have tried to make it relatively painless to extend Marked. If you are looking to add custom functionality, this is the place to start.
 
 <h2 id="use">marked.use()</h2>
 
-`marked.use(options)` is the recommended way to extend marked. The options object can contain any [option](/using_advanced#options) available in marked.
+`marked.use(extension)` is the recommended way to extend Marked. The `extension` object can contain any [option](/using_advanced#options) available in Marked:
 
-The `renderer` and `tokenizer` options can be an object with functions that will be merged into the `renderer` and `tokenizer` respectively.
 
-The `renderer` and `tokenizer` functions can return false to fallback to the previous function.
+```js
+const marked = require('marked');
+
+marked.use({
+  pedantic: false,
+  gfm: true,
+  breaks: false,
+  sanitize: false,
+  smartLists: true,
+  smartypants: false,
+  xhtml: false
+});
+```
+
+You can also supply an array of multiple `extension` objects.
+
+```
+marked.use([myExtension, extension2, extension3]);
```

Comment:
Do you think we should allow multiple parameters (i.e. `marked.use(extension1, extension2, etc)`) instead of an array? might be a little cleaner for the user and not any harder for marked to implement.

Reply:
Sure, if that seems like a better way to do it.

Reply:
Ok this is changed.